### ISSUE TITLE
Train Blueprints Become Disconnected

### ISSUE THREAD

**Issue by bcwhite-code** [nyurik/fatul#10]
If I import then export this blueprint, all of the train cars are un-joined.

```

0eNrtWd+P2jgQ/lcqP4dTfjjZBbV96N1VutdT3ypkmWTIWpvYke3AoRX/+42TAIGGArvtFtR9Wdae8cw34/nG2DyRWVFDpYW0ZPJERKqkIZOvT8SIXPLCzUleApmQQqWqVFYsgKw9ImQG/5FJsPb6qnZVOdWF0LbGGW+zttUY0d7KcH9lp5hynavRkudK9nSjZ3uh66lHQFphBbRxNYMVk3U5A40BeAPxeaRSBpcgCHSHZkbje/+Pu+D+LvbICodJhIGjKT4rgBUqF8aK1LDlg8BxqRZC5mQy54UBjygt0CdvraEVNJGqQmlnGv/4HsndPEUX0TgMYvwIaBIlYeTTpPE4a2Byl22XiIMIQm8oeQMh3EU/NIRqxZo42FyrklnNhWTGqopMrK7B7cACFymNzmRdFAPIo3ORJ8mVIafnIo/HV4Y8Pht5eGXIkzOZSuPrYCo2HpM+QFYXXefZwXbjwEt6Gs5FrlWN+SCfvkwobg5xSbGgdV3Z1kAXPsr/hXkNrvNhu86a8FsNdFBxDazrkFxmqNMN3AomLJS4EbW0jMtV34BDMBfaWPZtU1Y4XDe6xnJ3UMS+1/ni1uWHvCdNa/o5/o0qRDZqIv6FKLRKH8EOwAheFYas0wK4HsIxAANLzCLPoasgs6luLKJ3n9sSWnJh2aV1NEeOkoEgld4pCslTrHbksJsT6SPajnz/xCorStjpB/e4YOrCENKIDNiWEm3jcMZ2tKBW0ctIAWWF8E7tVip0Wgt7aodOfC/pPtwGlYBR7O8ePdi9D4NF1M/U9WCPbxh7csPY724YexDeMvjolsHfcqsJhnrN0VPua+ekM/qhrkZca7WcHgo2V8djgkNQ02cdnMNn5k/IoYZsP23+xVs+dHoHoX8z+Pe/R0SJfzvQfwyQXAPI70P56LjTvK0cZUymlvIXcuY1G88baQ5JM34jzTHSDN5ImneE/SvJ366AL7yUcATcEdLbv7L9BZWyJ+kglWUdcc4+GV/Ac3eNfWP5G8t/Y5a3vHxVkmdgLBaiW9a8xjClmVSs4vbh1MOM6w8vdX+Usn1Hx518bl+vXt7HdoDPfCP6Jyvgsp2quDEgc3D5tazSgCN7DqTLG+E+M5Pjb19tDU7bV2ar0kfnRUK6c9PMNo+CM+7+aX9vayfx6j3XqnszbMVRTxxtxeFGTHtiuhVHG3HcE8dbMd2Ik5442YrjJjQ8PDDw3Y+QHin4DItjQujoT67ffXE/FeDsArRp8hsn4ZiOxzENaBgGwXr9P6DS3u4=

```

**Comment by nyurik** [comment]
Thanks, fixed in b70420b44562f014cff47f1495a34ce86b46ebde

**Comment by nyurik** [comment]
P.S. This was due to yet another new field: `stock_connections`, which i documented in [here](https://wiki.factorio.com/Talk:Blueprint_string_format#v2.x_introduced_new_%22stock_connections%22_array_for_trains(?))
